client/my-sites/comments/comment/utils.js

```diff
@@ -15,4 +15,5 @@ export const getMinimumComment = comment => ( {
 	isLiked: get( comment, 'i_like' ),
 	postId: get( comment, 'post.ID' ),
 	status: get( comment, 'status' ),
+	can_moderate: get( comment, 'can_moderate' ),
```

Comment:
Since we're only using this in Calypso, do you mind camel-casing it?
(and update the `comment-list/index.jsx` line too)

Reply:
Could we do this in a follow up PR? I think we'd like to do the rename at the data-layer level then in all locations. (mismatches between snake and camel can be confusing if it's done inconsistently)